webview/src/shared/components/codeSlider/CodeSlider.tsx

```diff
@@ -0,0 +1,48 @@
+import cx from 'classnames'
+import React, { useState } from 'react'
+import styles from './styles.module.scss'
+import { CodeBlock, CodeBlockProps } from '../codeBlock/CodeBlock'
+
+interface CodeBlockWithTitle extends CodeBlockProps {
+  title: string
+}
+
+interface CodeSliderProps {
+  codeBlocks: CodeBlockWithTitle[]
+}
+
+export const CodeSlider: React.FC<CodeSliderProps> = ({ codeBlocks }) => {
+  const [active, setActive] = useState(0)
```

Comment:
Can we change the tabs on hover (cursor, highlight text, add border on hover for example)? 

Probably minor, but there's also this weird kind of jumping that happens when switching tabs:

https://user-images.githubusercontent.com/43496356/208214297-ef4fd517-8e8e-4986-b522-2db9ea457157.mov

Reply:
The data is centered that's why there is some jumping. I was thinking about it this weekend. I can probably make the code block the same height (through a wrapper). That might look weirder as it will be off-centered, but maybe it won't.